### PR TITLE
Bitly docs

### PR DESCRIPTION
**Pull request by Shraddha2104**
Hi,
✨ Here are the basic instructions for the Bitly node.

### CODE REVIEW

File: docs/nodes/nodes-library/nodes/Bitly/README.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+- Link
+	- Create a link
+	- Get a link
+	- Update a link
+
+
+## Example Usage
+
+This workflow demonstrates creating a new link. You can also find the [workflow](https://n8n.io/workflows/442) on this website. This example usage workflow uses the following two nodes.
```
Comment (ivov) [inline docs/nodes/nodes-library/nodes/Bitly/README.md:23]: Another option: `This workflow shows you how to create a new link.` 

This other version has been approved in other merges.

File: docs/nodes/nodes-library/nodes/Bitly/README.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+The final workflow should look like the following image.
+
+![A workflow with the Bitly node](./workflow.png)
+
+### 1. Start node
+
+The start node exists by default when you create a new workflow.
+
+### 2. Bitly Node
```
Comment (ivov) [inline docs/nodes/nodes-library/nodes/Bitly/README.md:36]: Lowercase `node`

File: docs/nodes/nodes-library/nodes/Bitly/README.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+1. First, ensure that you have entered authentication credentials through [this tutorial](../../../credentials/Bitly/README.md).
+2. To create a link, select the *link* option under the *Resource* field.
```
Comment (ivov) [inline docs/nodes/nodes-library/nodes/Bitly/README.md:39]: Uppercase `*Link*`

Comment (ivov) [reply docs/nodes/nodes-library/nodes/Bitly/README.md:39]: `in the *Resource* field`

File: docs/nodes/credentials/Bitly/README.md [hunk 4]
```diff
@@ -1,38 +0,0 @@
...
-
-1. Open your Bitly dashboard.
-2. Click on your account button in the top right.
-3. Click on "Profile Settings".
-4. Click on "Generate Access Token".
-5. Enter password.
-6. Use the access token in your Bitly node credentials in n8n.
-
-
-![Alt Text](https://i.imgur.com/sE4kUSa.gif)
```
Comment (ivov) [inline docs/nodes/credentials/Bitly/README.md:35]: Should this whole file be deleted? Or we can just comment out the OAuth2 section? Please double check.

**Review by ivov — CHANGES_REQUESTED** [review]

File: docs/nodes/nodes-library/nodes/Bitly/README.md (hunk 1 above)
Comment (Shraddha2104) [reply docs/nodes/nodes-library/nodes/Bitly/README.md:23]: I'll change it accordingly.

**Comment by Shraddha2104** [comment]
Submitted to maintainers

**Pull request closed without merging**